src/main/java/org/gridsuite/filter/server/FilterController.java

```diff
@@ -86,11 +86,11 @@ public FilterController(FilterService service) {
         return ResponseEntity.ok().build();
     }
 
-    @PostMapping(value = "/filters/metadata")
+    @GetMapping(value = "/filters/metadata")
     @Operation(summary = "get filters metadata")
     @ApiResponses(value = {@ApiResponse(responseCode = "200", description = "filters metadata"),
         @ApiResponse(responseCode = "404", description = "The filters don't exist")})
-    public ResponseEntity<List<FilterAttributes>> getFiltersMetadata(@RequestBody List<UUID> ids) {
+    public ResponseEntity<List<FilterAttributes>> getFiltersMetadata(@RequestParam("ids") List<UUID> ids) {
```

Comment:
So now we are fine with giving a list in request param ?

Reply:
Necessary for the controls in the gateway